script/convert_mesh.py

```diff
@@ -117,6 +121,26 @@ def main():
 
         mesh = domain.mesh
 
+    if options.tri_tetra > 0:
+        conns = None
+        for k, new_desc in [('3_8', '3_4'), ('2_4', '2_3')]:
+            if k in mesh.descs:
+                conns = mesh.get_conn(k)
+                break
+
+        if conns is not None:
+            nelo = conns.shape[0]
+            output('initial mesh: %d elements' % nelo)
+
+            new_conns = elems_q2t(conns)
+            nn = new_conns.shape[0] // nelo
+            new_cgroups = nm.repeat(mesh.cmesh.cell_groups, nn)
+
+            output('... %d elements' % new_conns.shape[0])
```

Comment:
With output, `...` usually denotes un-indent one level - maybe better would be to have simply:
```
output('new mesh: %d elements' % new_conns.shape[0])
```